App/Seller/Sources/Seller/UI/MainView.swift

```diff
@@ -2,33 +2,45 @@
 import SwiftUI
 
 struct MainView: View {
-
+    
     var body: some View {
         TabView {
             Group {
-                MyOffersView()
-                    .tabItem {
-                        Label("Moje oferty", systemImage: "list.bullet")
-                    }
-                   
-                MyTransactionsView()
+                MessagesView()
                     .tabItem {
-                        Label("Transakcje", systemImage: "dollarsign.square")
+                        TabBarIcon(imageName: "messageTabIcon", titleName: "Wiadomości")
                     }
-                   
-                MessagesView()
+                MyOrdersView()
                     .tabItem {
-                        Label("Wiadomości", systemImage: "message")
+                        TabBarIcon(imageName: "orderTabIcon", titleName: "Zamówienia")
                     }
-
-                MySettingsView()
+                MyAccountView()
                     .tabItem {
-                        Label("Ustawienia", systemImage: "slider.horizontal.3")
+                        TabBarIcon(imageName: "accountTabIcon", titleName: "Konto")
                     }
             }
         }
         .navigationBarBackButtonHidden(true)
     }
+    
+    private struct TabBarIcon: View {
+        private let imageName: String
+        private let titleName: String
+        
+        init(imageName: String, titleName: String) {
+            self.imageName = imageName
+            self.titleName = titleName
+        }
+        
+        var body: some View {
+            VStack(spacing: 0) {
+                Image(imageName)
+                    .padding(.bottom, 10)
+                Text(titleName)
+                    .design(typography: .label(weight: .regular, size: 12))
```

Comment:
`weight` has a default value so you can skip it here :)

Maybe typography should have a case for `tabBarLabel`? I'm worrying that later there will be a lot of constant like this `12` here all over the place :)